web/test/pages/profile/profile-owning.test.tsx

```diff
@@ -139,25 +133,23 @@ describe("profile - owning existing business", () => {
         businessPersona: "OWNING",
         industryId: "generic",
         legalStructureId: "limited-liability-company",
-        municipality: undefined,
       }),
       onboardingFormProgress: "COMPLETED",
-      formationData: formationWithUndefinedMunicipality,
     });
-    const newark = generateMunicipality({ displayName: "Newark" });
+    const randomMuni = generateMunicipality({ displayName: "Newark" });
```

Comment:
pebble - I don't love the abbreviation here. Would personally prefer randomMunicipality

Reply:
I made the change across the board